src/node_crypto.cc

```diff
@@ -3797,9 +3797,7 @@ void Hmac::HmacDigest(const FunctionCallbackInfo<Value>& args) {
 
   enum encoding encoding = BUFFER;
   if (args.Length() >= 1) {
-    encoding = ParseEncoding(env->isolate(),
-                             args[0]->ToString(env->isolate()),
-                             BUFFER);
+    encoding = ParseEncoding(env->isolate(), args[0], BUFFER);
```

Comment:
> Is it okay to do that in a separate PR or will that create too much organizational overhead?

That’s your call… you can also maintain multiple commits in a single PR, which may or may not be easier for *you*. :) I don’t think there are other open pull requests for this code that would generate merge conflicts, and everything we talked about (except maybe dropping `toString()` support) semver-patch.